### DISCUSSION TITLE
Categories for People Directly Shared to Repo

### DISCUSSION THREAD

**Discussion by BD103** [community/community#3125] [Discussions]
Having the ability so that not just anyone can create a discussion in a category would be amazing.

**Comment by evi-liu** [comment]
Do you have specific examples of use cases?

**Reply by BD103** [reply]
For instance in a team, there can be a "rank" of sorts. Owner, Maintainer, Collaborator, Contributor, etc. You can apply a discussion to be "Collaborator and Higher Only", so that regular Github members cannot comment. This would be especially handy for an announcements. There can also be certains rules, like "Emote Only". Maybe also a way to apply this to a broad category in discussions. For instance, a "Notice" category that only the Owner can post to. Others can subscribe to categories to get notified about new posts.
